### PR TITLE
Added oschecks-pacemaker_host_check documentation

### PR DESCRIPTION
**Pull request by jbadiapa**
This part of the documentation depends on the change/review  https://review.openstack.org/#/c/359943/

### CODE REVIEW

File: availability-monitoring-oschecks.txt [hunk 1]
```diff
@@ -21,6 +21,10 @@ CEPH UNKNOWN: 2016-08-19 15:42:23.883982 7f07f37c3700 -1 monclient(hunting): ERR
 2016-08-19 15:42:23.883983 7f07f37c3700  0 librados: client.admin initialization error (2) No such file or directory
 Error connecting to cluster: ObjectNotFound
 
+NOTE: The oschecks-pacemaker_host_check command is using either "pcs status" or "crm_mon -1" to use them you should be able to execute the 
+command as that users
```
Comment (paramite) [inline availability-monitoring-oschecks.txt:25]: Shouldn't this be on the same line a previous?

**Comment by paramite** [comment]
+1

File: centralised-logging.txt [hunk 2]
```diff
@@ -14,6 +14,7 @@ First thing we will need to open the firewall on the machine to allow connection
 
  firewall-cmd --zone=public --add-port=4000/tcp --permanent
  firewall-cmd --zone=public --add-service=http --permanent
+ firewall-cmd --zone=public --add-port=5601/tcp --permanent #Optional (to use kibana directly)
```
Comment (mrunge) [inline centralised-logging.txt]: Could you make it more clear, that this is only required, if kibana is used directly?

File: centralised-logging.txt [hunk 3]
```diff
@@ -79,33 +80,20 @@ Check the journal for fluentd and ensure it has no errors at start
 
  journalctl -u fluentd -l -f
 
-Finally we will configure kibana to point to our elasticsearch instace. Create the file /etc/httpd/conf.d/kibana3.conf and place the following content inside
+Finally we will configure kibana to point to our elasticsearch instace. Create the file /etc/httpd/conf.d/kibana.conf and place the following content inside
```
Comment (mrunge) [inline centralised-logging.txt]: same here: a note on this is going to configure reverse proxy and opening the firewall to use kibana directly is not necessary.

**Review by mrunge — COMMENTED** [review]
looks ok to me, I would like to make that step (use kibana directly) more clear.

**Review by mrunge — APPROVED** [review]

**Comment by paramite** [comment]
We should really keep PR focused. This PR contains not only pacemaker oscheck changes.

**Comment by mrunge** [comment]
should we continue with this here?

**Comment by paramite** [comment]
I would say that @jbadiapa should :)

**Comment by jbadiapa** [comment]
This pull request was split in two #13 & #14

**Pull request closed without merging**